tortoise/verifying_tortoise_test.go

```diff
@@ -3059,3 +3065,106 @@ func BenchmarkTortoiseLayerHandling(b *testing.B) {
 		}
 	}
 }
+
+func randomBytes(t *testing.T, size int) []byte {
+	data, err := crypto.GetRandomBytes(size)
```

Comment:
why is it crypto? not a big deal, but test code should preferably use randomness from math package

Reply:
changed to math.